docs/_plugins/sidebar_item.rb

```diff
@@ -4,7 +4,14 @@ def sidebar_item_link(item)
       pageID = @context.registers[:page]["id"]
       itemID = item["id"]
       href = item["href"] || "/react/docs/#{itemID}.html"
-      className = pageID == itemID ? ' class="active"' : ''
+      classes = []
+      if pageID == itemID
+        classes.push("active")
+      end
+      if item["href"]
+        classes.push("external")
+      end
+      className = classes.size ? "class=\"#{classes.join(' ')}\"" : "";
```

Comment:
`bundle exec`? It shouldn't really be a bundler issue though because 0 isn't falsey in Ruby.